packages/story-editor/src/app/story/utils/isEmptyStory.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { shallowEqual } from '@googleforcreators/react';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_PAGE_BACKGROUND_COLOR } from '../../../elements/utils/createPage';
+
+const isEmptyStory = (pages) => {
```

Comment:
Just bringing this up in case you haven't seen it yet - last year we added some new story triggers, this feels a lot like this one here: https://github.com/GoogleForCreators/web-stories-wp/blob/main/packages/story-editor/src/app/story/storyTriggers/storyEvents/onInitialElementAddedRegister.js

Reply:
`isNewStory` does just the first 2 checks of `isEmptyStory` which are quite critical for this PR, not sure if you can replace `isNewStory` with `isEmptyStory`, if you change the background color or add bg media - is the story still "new"? for me it's not, so maybe we can replace it. WDYT?

Reply:
> Also curious if when in RTL the menu should stay centered?
> <img alt="Screen Shot 2022-02-07 at 10 50 59 AM" width="616" src="https://user-images.githubusercontent.com/10720454/152844267-27fff733-0885-43d8-9b0d-809c2cd9252d.png">

@felipebochehin87 good find by Brittany, this RTL is sneaky little one 😉

Reply:
for sure, i just wanted to mention the structure's existence should it be of use for you here.